roles/edpm_nova/molecule/default/verify.yml

```diff
@@ -186,3 +186,29 @@
       ansible.builtin.assert:
         that:
           - "not nova_host_specific_conf.stat.exists"
+
+    - name: Disable nova nvme cleaner
+      ansible.builtin.include_role:
+        name: osp.edpm.edpm_nova
+      vars:
+        edpm_nova_enable_nvme_cleaner: false
+
+    - name: Assert nvme cleaner container is unavailable or deleted
+      become: true
+      ansible.builtin.command: podman inspect nova_nvme_cleaner
+      register: nvme_cleaner_exists
+      # this will fail as container is deleted
+      failed_when: false
```

Comment:
OK this is just collecting the data. The asserts are below checking the exit code